Movieee/Movieee/SceneDelegate.swift

```diff
@@ -5,8 +5,14 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
 
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-        guard let _ = (scene as? UIWindowScene) else { return }
+        if let rvc = self.window?.rootViewController {
+            let ud = UserDefaults.standard
+            if ud.bool(forKey: "hasLaunched") == true {
```

Comment:
```suggestion
            if ud.bool(forKey: "hasLaunched") {
```
Hard code, để ra Constant